Created SupplierRegisterPage

rares2k opened the pull request:
Created SupplierRegisterPage, if there are more containers that need to be put in the column, the column can be changhed in a listview.

rares2k committed:
Created SupplierRegisterPage

Created SupplierRegisterPage, if there are more containers that need to be put in the column, the column can be changhed in a listview.

CimpeanAndreea commented on lib/main.dart:
```diff
@@ -22,7 +24,7 @@ class _MyAppState extends State<MyApp> {
   Widget build(BuildContext context) {
     return MaterialApp(
-        home: LoginPage()
+        home: SupplRegisterPage()
```
Please do not use abbreviations and revert, will be handled when navigation is implemented.

CimpeanAndreea commented on lib/register_supplier/suppl_registerpage_form.dart:
```diff
@@ -0,0 +1,145 @@
+                    margin: EdgeInsets.only(right: 30),
+                    child: DefaultTextStyle(
+                      style: TextStyle(
+                        color: Color.fromARGB(255, 27, 21, 18), 
```
Define a constant for the color

CimpeanAndreea commented on lib/register_supplier/suppl_registerpage_form.dart:
```diff
@@ -0,0 +1,145 @@
+              margin: EdgeInsets.fromLTRB(0, 5, 0, 5),
+              child: DefaultTextStyle(
+                style: TextStyle(
+                  color: Color.fromARGB(255, 27, 21, 18), 
```
Define a constant for the color

CimpeanAndreea commented on lib/register_supplier/suppl_registerpage_form.dart:
```diff
@@ -0,0 +1,145 @@
+              margin: EdgeInsets.fromLTRB(0, 0, 0, 20),
+              child: DefaultTextStyle(
+                style: TextStyle(
+                  color: Color.fromARGB(255, 27, 21, 18), 
```
Constant

CimpeanAndreea commented on lib/register_supplier/suppl_registerpage_form.dart:
```diff
@@ -0,0 +1,145 @@
+                    child: TextField(
+                      decoration: InputDecoration(
+                        border: OutlineInputBorder(),
+                        hintText: "Unic code"
```
Are we sure this is called "Unic Code"? Seems odd to me

CimpeanAndreea commented on lib/register_supplier/suppl_registerpage_form.dart:
```diff
@@ -0,0 +1,145 @@
+                      color: Colors.white
+                    ),
+                  ),
+                  color: Color.fromARGB(255, 42, 157, 143),
```
Constant

CimpeanAndreea commented on lib/register_supplier/suppl_registerpage_image.dart:
```diff
@@ -0,0 +1,23 @@
+        child: DecoratedBox(
+          decoration: BoxDecoration(
+            image: DecorationImage(
+              image: AssetImage("assets/registerpageimage.png"),
```
Use/create constant for the path

CimpeanAndreea commented on lib/register_supplier/suppl_registerpage_image.dart:
```diff
@@ -0,0 +1,23 @@
+  @override
+  Widget build(BuildContext context) {
+    return Expanded(
+      flex: 90, //avem valori asa de mari aici pentru a face bara colorata din mijloc mica
```
Remove these comments

CimpeanAndreea commented on lib/register_supplier/suppl_registerpage_logo.dart:
```diff
@@ -0,0 +1,20 @@
+      right: 0,
+      top: screenHeight * 0.016,
+      child: Image.asset(
+        "assets/bluelogo.png",
```
Use/create constant for path

CimpeanAndreea commented on lib/register_supplier/suppl_registerpage_suppl.dart:
```diff
@@ -0,0 +1,40 @@
+            width: double.infinity,
+            decoration: const BoxDecoration(
+                image: DecorationImage(
+                    image: AssetImage("assets/registerpagegradient.png"),
```
Use/create constant for path

CimpeanAndreea commented on lib/register_supplier/suppl_registerpage_suppl.dart:
```diff
@@ -0,0 +1,40 @@
+                            child: Container(
+                                height: double.infinity,
+                                width: double.infinity,
+                                color: Color.fromARGB(255, 42, 157, 143))),
```
Constant

CimpeanAndreea reviewed (commented):
Please change everywhere from "suppl" to "supplier" in code and file names as well

rares2k committed:
Fixed the problems

Changed the image paths with constants, updated the constants file and changed the name of the classes and files.

rares2k committed:
Merge branch 'master' into feature/supplier_register

rares2k committed:
Merge branch 'master' into feature/supplier_register
